quickstart/armv7a9-zynq7000-zturn.md

```diff
@@ -0,0 +1,245 @@
+  If your output is like in the screenshot above, the console (`UART` in the evaluation board) is on the `USB0` port.
+
+- When the board is connected to your host-pc, open serial port in terminal using picocom and type the console port
+  (in this case USB0)
+
+  ```bash
+  picocom -b 115200 --imap lfcrlf /dev/ttyUSB0
+  ```
+
+  To use picocom without sudo privileges run this command and then restart:
+
+  ```bash
+  sudo usermod -a -G tty <yourname>
+  ```
+
+- You should see such output:
+  
+  <img src="_images/zynq7000-zturn-picocom.png" width="700px">
+
+  <details>
+  <summary>How to get picocom (Ubuntu 20.04)</summary>
+
+  ```bash
+  sudo apt-get update && \
+  sudo apt-get install picocom
+  ```
+
+  </details>
+  </br>
+
+You can leave the terminal with the serial port open, and follow the next steps.
+
+## Flashing the Phoenix-RTOS system image
+
+At first before any flashing, you need to enter Phoenix-RTOS loader (plo).
+
+### Entering Phoenix-RTOS loader (plo)
+
+Press RESET button (`K2`) to restart the chip.
+
+If flash memory doesn't contain system image, booting process will stop at plo level, you should see:
+
+<img src="_images/zynq7000-zturn-magic.png" width="400px">
+
+This error just says that no system image is on the flash, just plo.
+
+Otherwise, in order to get into plo you need to press any key within 500ms. If you don't do that, plo will load system
+and start psh, but we cannot flash from there. Output when you enter psh:
+
+<img src="_images/zynq7000-zturn-psh.png" width="700px">
+
+Restart the chip with RESTART button `K2` and try again. Output of successful enter to plo:
+
+<img src="_images/zynq7000-zturn-plo.png" width="700px">
+
+If you want to flash the system image please follow the next steps.
+
+### Copying image to flash memory using PHFS (phoenixd)
+
+To flash the disk image, first, you need to verify on which port plo USB device has appeared. You can check that using
+`ls` as follow:
+
+```bash
+ls -l /dev/serial/by-id
+```
+
+<img src="_images/zynq7000-ls-2.png" width="700px">
+
+To share disk image to the bootloader, `phoenixd` has to be launched with the following arguments (choose suitable
+ttyACMx device, in this case, ttyACM0):
+
+```bash
+cd _boot/armv7a9-zynq7000-zturn
+```
+
+```bash
+sudo ./phoenixd -p /dev/ttyACM0 -b 115200 -s .
+```
+
+<img src="_images/zynq7000-phoenixd.png" width="700px">
+
+Before flashing, good practise is to erase older file system on flash memory (this is done to avoid errors).
+
+### Erasing the area intended for file system
+
+It's needed to erase sectors that will be used by `jffs2` file system as we place in the `phoenix.disk`
+ only the necessary file system content, not the whole area intended for it.
+Without erasure `jffs2` may encounter data from the previous flash operation and errors
+ during the system startup may occur.
+That's why we have run erase using plo command specific to `jffs2` file system:
+
+```bash
+jffs2 -d 2.0 -e -c 0x400:0x400:0x2000:16
+```
```

Comment:
Please use plaintext when presenting commands from our shell/loader. When using `bash`, only commands present in bash will be colored.

```suggestion
```plaintext
jffs2 -d 2.0 -e -c 0x400:0x400:0x2000:16
```
```